package.json

```
@@ -1,77 +1,79 @@
 {
-	"name": "branch-in-window-title",
-	"displayName": "Branch in Window Title",
-	"description": "Visual Studio Code extension that shows the current Git branch in the window title. This can be used to automate time allocation to issue tracking systems via WiseTime.",
-	"publisher": "wisetime",
-	"version": "0.1.4",
-	"license": "Apache-2.0",
-	"repository": {
-		"type": "git",
-		"url": "https://github.com/wisetime-io/vscode-branch-in-window-title.git"
-	},
-	"engines": {
-		"vscode": "^1.73.0"
-	},
-	"categories": [
-		"Other"
-	],
-	"keywords": [
-		"git",
-		"branch",
-		"window",
-		"title",
-		"wisetime"
-	],
-	"activationEvents": [
-		"onStartupFinished"
-	],
-	"contributes": {
-		"configuration": {
-			"title": "Branch in Window Title",
-			"properties": {
-				"branchInWindowTitle.branchPollingInterval": {
-					"type": [
-						"number",
-						"null"
-					],
-					"default": 3000,
-					"description": "Specifies the Git branch polling interval in milliseconds."
-				},
-				"branchInWindowTitle.branchTemplate": {
-					"type": "string",
-					"default": "${separator}[Branch: $branchName]",
-					"description": "Defines branch name appearance in window title. Use '$branchName' as placeholder."
-				},
-				"branchInWindowTitle.branchNameIsPrefix": {
-					"type": "boolean",
-					"default": false,
-					"description": "Prevent conflicts with time tracking software that thinks the application name always in window title after the last dash."
-				}
-			}
-		}
-	},
-	"main": "./out/extension.js",
-	"scripts": {
-		"vscode:prepublish": "npm run compile",
-		"compile": "tsc -p ./",
-		"lint": "eslint src --ext ts",
-		"watch": "tsc -watch -p ./",
-		"pretest": "npm run compile && npm run lint",
-		"test": "node ./out/test/runTest.js"
-	},
-	"devDependencies": {
-		"@types/glob": "^7.2.0",
-		"@types/mocha": "^9.1.0",
-		"@types/node": "^12.11.7",
-		"@types/text-encoding": "0.0.36",
-		"@types/vscode": "^1.66.0",
-		"@typescript-eslint/eslint-plugin": "^5.18.0",
-		"@typescript-eslint/parser": "^5.18.0",
-		"eslint": "^8.12.0",
-		"glob": "^7.2.0",
-		"mocha": "^9.2.2",
-		"text-encoding": "^0.7.0",
-		"typescript": "^4.6.3",
-		"vscode-test": "^1.6.1"
-	}
+    "name": "branch-in-window-title",
+    "displayName": "Branch in Window Title",
+    "description": "Visual Studio Code extension that shows the current Git branch in the window title. This can be used to automate time allocation to issue tracking systems via WiseTime.",
+    "publisher": "wisetime",
+    "version": "0.1.5",
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/wisetime-io/vscode-branch-in-window-title.git"
+    },
+    "engines": {
+        "vscode": "^1.70.0"
```

Can you revert `package.json` formatting tabs instead of spaces? It's hard to see actual changes.
Also, can you revert this change:
```suggestion
        "vscode": "^1.73.0"
```

---

I've pushed `.editorconfig` to avoid future confusions about formatting of those files.

What to VSCode version base, it was changed around the time of #17 and I frankly don't understand the reasoning.
This unwarranted change excluded people using Windows 7 from benefitting by these updates.